tests/network/vmi_networking.go

```diff
@@ -728,17 +709,17 @@ var _ = Describe(SIG("[rfe_id:694][crit:medium][vendor:cnv-qe@redhat.com][level:
 				Expect(err).ToNot(HaveOccurred())
 
 				By("Wait for VMIs to be ready")
+				vmi = libwait.WaitUntilVMIReady(vmi, console.LoginToAlpine)
 				anotherVmi = libwait.WaitUntilVMIReady(anotherVmi, console.LoginToAlpine)
-
-				vmi = libwait.WaitUntilVMIReady(vmi, console.LoginToFedora)
 			})
 
 			DescribeTable("should have the correct MTU", func(ipFamily k8sv1.IPFamily) {
 				libnet.SkipWhenClusterNotSupportIPFamily(ipFamily)
 
 				if ipFamily == k8sv1.IPv6Protocol {
-					// IPv6 address is configured via DHCP6 in this test and not via cloud-init
-					Expect(configureIpv6(vmi)).To(Succeed(), "failed to configure ipv6  on server vmi")
+					// IPv6 address is configured via DHCP6 in this test and not statically.
```

Comment:
Is the DHCPv6 client enabled by default in this version of Alpine?
In the cloud-init network config provided by `libnet.WithMasqueradeNetworking()`, dhcpv6 is not explicitly enabled and the cloud-init default for DHCPv6 is disabled by default [1] (at least in contemporary versions).

[1] https://cloudinit.readthedocs.io/en/latest/reference/network-config-format-v2.html#dhcp6-bool

Reply:
I’m not clear how cloud-init is relevant anymore. I removed it completely.

And as the ping passes, my assumption about the default being dhcp enabled like with Fedora stands.